kotlin/gradlew.bat

```diff
@@ -1,3 +1,19 @@
+@rem

```

Comment:
Please ignore changes in this file. These scripts are auto generated by Android Studio when importing, and are only usable inside of Android studio. I'm only including them in the PR, so that when a candidate imports the project into Android studio they see a smaller (hopefully zero) local git changes.